lib/ssl/doc/src/ssl.xml

```diff
@@ -1462,14 +1462,31 @@ fun(srp, Username :: binary(), UserState :: term()) ->
 	<name name="server_session_tickets"/>
 	<desc>
 	  <p>Configures the session ticket functionality. Allowed values are <c>disabled</c>,
-	  <c>stateful</c> and <c>stateless</c>.</p>
+	  <c>stateful</c>, <c>stateless</c>, <c>stateful_with_cert</c>, <c>stateless_with_cert</c>.</p>
 	  <p>If it is set to <c>stateful</c> or
```

Comment:
Maybe here it would make more sense to write. If it is not disabled, the session resumption with pre-shared keys.

Reply:
I agree, fixed